src/pages/api/workspaces/list.ts

```diff
@@ -5,18 +5,40 @@ import { type Workspace, workspace } from "lib/zod/index.ts"
 
 export default withRouteSpec({
   methods: ["GET", "POST"],
-  auth: "cst_ak_pk",
```

Comment:
Can we remove this from the rest of the project now?

Reply:
6cb24e1b82b9fc3c14783c2e17992a051ff9f2df